packages/next/next-server/server/next-server.ts

```diff
@@ -823,7 +823,9 @@ export default class Server {
       if (revalidate) {
         res.setHeader(
           'Cache-Control',
-          `s-maxage=${revalidate}, stale-while-revalidate`
+          revalidate < 0
```

Comment:
This differs than the behavior in serverless loader, which would be `revalidate <= 0`. 

Any specific reason?

Reply:
Wasn't updated in serverless-loader correct, should be updated now